sound/soc/sof/debug.c

```diff
@@ -126,6 +127,19 @@ int snd_sof_dbg_init(struct snd_sof_dev *sdev)
 				map->name);
 	}
 
+#ifdef DEBUG
+	/* enable debug mode for DEBUG build */
+	sdev->debug_mode = 1;
+#else
+	sdev->debug_mode = 0;
```

Comment:
there might not be need for this if we bzero sdev on allocation ?

Reply:
If we bzero sdev, then it is not needed.
I will check that.
Thanks for reminding.